halibot/halobject.py

```diff
@@ -42,21 +42,48 @@ def init(self):
 	def shutdown(self):
 		pass
 
+	def apply_filter(self, dest):
+		fl = self._hal.config.get("filters")
+		if not fl:
+			return dest # Filters not enabled/configured
+
+		name = dest.split("/")[0]
+
+		ib = fl.get("inbound", {})
+		ret = "/".join(ib[name] + [dest]) if name in ib else dest
```

Comment:
`ret = "/".join(ib.get(name, []) + [dest])` is shorter but maybe more opaque

Reply:
Ahhhh this was the one liner I knew was in there somewhere. Opague, but cleaner.